soc/arm/silabs_exx32/efr32mg21/soc.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 TriaGnoSys GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Board configuration macros for the EFR32MG21 SoC
+ *
+ */
+
+#ifndef _SOC__H_
```

Comment:
`ZEPHYR_SOC_ARM_SILABS_EXX32_EFR32MG21_SOC_H_` instead?

Reply:
@parthitce Ok done, thanks for the suggestion